curation/models.py

```diff
@@ -213,8 +213,9 @@ def __getattr__(self, attr):
 
         # We would get an infinite loop if self.field_overrides[attr] == attr
         if attr in self.field_overrides and self.field_overrides[attr] != attr:
-            val = getattr(self, self.field_overrides[attr])
-            if val or isinstance(val, bool):
+            sentinel_value = object()
+            val = getattr(self, self.field_overrides[attr], sentinel_value)
```

Comment:
for that matter, this doesn't actually fix the problem, right? If you were going to go this route, you would need

```python
val = getattr(self, self.field_overrides[attr]) or sentinel_value
```

but that doesn't work when the value is boolean. This is the wrong fix, because it doesn't check whether there's actually a foreign object whose properties are being proxies.